nva-datamodel-java/src/main/java/no/unit/nva/model/ImportDetail.java

```diff
@@ -0,0 +1,5 @@
+package no.unit.nva.model;
+
+import java.time.Instant;
+
+public record ImportDetail(Instant date, ImportSource source) { }
```

Comment:
I would also add following annotation to this record:
`@JsonTypeInfo(use = JsonTypeInfo.Id.NAME, property = "type")`

So it serializes like this:
```
{
    "type" : "ImportDetail",
    "date" : "2024-06-11T07:01:51.427190Z",
    "source" : "BRAGE"
  }
```
Instead of:
```
{
    "date" : "2024-06-11T07:01:51.427190Z",
    "source" : "BRAGE"
  }
```

Reply:
Good points! Will do